### DISCUSSION TITLE
After fine-tuning, struggling to run the model on the test set

### DISCUSSION THREAD

**Discussion by ishikaj12** [openai/whisper#1384] [Q&A]
Hello! I've been working on fine-tuning Whisper for a low-resource language, and have been using this blogpost / accompanying Colab to do so: https://huggingface.co/blog/fine-tune-whisper

I'm running into an issue with how to run the trained model on my test set. I save the model and processor: 
<img width="535" alt="Screen Shot 2023-05-23 at 12 00 51 PM" src="https://github.com/openai/whisper/assets/77066462/ea54ba1f-ddac-4bea-ad09-87b5f0465465">

Run the training and everything works okay. Then I load the pretrained model and processor which also works fine: 
<img width="1153" alt="Screen Shot 2023-05-23 at 1 29 26 PM" src="https://github.com/openai/whisper/assets/77066462/4acb9794-3a4c-4bdd-827b-e774f01464a6">

Then I try to just take the first example of the test set, run it through the model and take the argmax of the logits to retrieve the predicted token ids (and then will decode it) 
<img width="891" alt="Screen Shot 2023-05-23 at 1 34 58 PM" src="https://github.com/openai/whisper/assets/77066462/c6149573-d3ce-4ff7-84e1-44f6e276b7aa">

But that keeps giving me this error, and I have not been able to fix it at all: 
<img width="1256" alt="Screen Shot 2023-05-23 at 1 35 10 PM" src="https://github.com/openai/whisper/assets/77066462/3100801e-1d1b-4835-9c5a-2f9074eaf23d">

Would appreciate any guidance or help, thank you so much!
